public/js/app/components/AnalysisMenu.jsx

```diff
@@ -321,34 +313,43 @@ function AnalysisMenu(props) {
       {/* ************************************************************************************ */}
       {/*                               SEMESTER DROPDOWN                          */}
       {/* ************************************************************************************ */}
-      <Dropdown isOpen={dropdownOpen} toggle={toggleDropdown} className="select-semester">
-        <div className="inline-flex padding-top-30">
+
+      <div className="col-4 nopadding">
+      <div className="inline-flex padding-top-30">
           <h3> {translate.select_semester} </h3>
           <InfoButton addClass="padding-top-30" id="info_select_semester" textObj={translate.info_select_semester} />
+      </div>
+      <form>
+        <div className="form-group">
+          <div className="form-select">
+            <div className="select-wrapper">
+              <select 
+              className="form-control"
+              id="semesterDropdownControl" 
+              aria-label={translate.select_semester} 
+              onChange={handleSelectedSemester}>
+                <option>
+                  {semester && semester > 0 && !firstVisit
+                  ? `${translate.course_short_semester[semester.toString().match(/.{1,4}/g)[1]]} 
+                  ${semester.toString().match(/.{1,4}/g)[0]}`
+                  : translate.select_semester}
+                </option>
```

Comment:
- for a default value, you can try to use an option `defaultValue` inside `<select>` tag, ex, `defaultValue={semester && semester > 0 && !firstVisit ? semester : translate.select_semester}`
- it means this `option` tag should get `value={translate.select_semester}`
- remove inside this option condition with`${translate.course_short_semester[semester.toString().match(/.{1,4}/g)[1]]} ${semester.toString().match(/.{1,4}/g)[0]}`.

Reply:
Very interesting, learning so much now. I will make the changes in a bit.